README.md

```diff
@@ -8,15 +8,17 @@ An action to automate your branch update process. This action opens a pull reque
  - Open a pull request whenever there are changes to a branch you want to observe.
  - Before opening a new pull request, check if there is already an open one with the same purpose. If so, a new pull request is not opened.
  - If a pull request is already open and there are changes to the observed branch, those changes are automatically included in the pull request already open.
+ - If `label` is provided and doesn't exist in the repository, action creates missing label. 
 
 ## Inputs
-|     Name     |     Type    |   Required  |    Description    |
-| --------------- |    :----:   |    :----:   | ----------------- |
-| GITHUB_TOKEN |    String   |     true    | User token to be associated with this pull request. |
-| SOURCE_BRANCH | String | true | Branch where you implemented your new code. |
-| DESTINATION_BRANCH | String | true | Branch you want to send your new code. |
-| PULL_REQUEST_TITLE | String | false | Standard title for your update pull request. Default is: `"update: sync ${DESTINATION_BRANCH} with ${SOURCE_BRANCH}` |
-| PULL_REQUEST_BODY | String | false | Content with description for your automated pull. Default is: `"This is an automatic Pull Request to keep ${DESTINATION_BRANCH} up to date with ${SOURCE_BRANCH}! 🔄"` |
+| Name               |  Type   | Required | Description                                                                                                                                                            |
+|--------------------|:-------:|:--------:|------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| GITHUB_TOKEN       | String  |   true   | User token to be associated with this pull request.                                                                                                                    |
+| SOURCE_BRANCH      | String  |   true   | Branch where you implemented your new code.                                                                                                                            |
+| DESTINATION_BRANCH | String  |   true   | Branch you want to send your new code.                                                                                                                                 |
+| PULL_REQUEST_TITLE | String  |  false   | Standard title for your update pull request. Default is: `"update: sync ${DESTINATION_BRANCH} with ${SOURCE_BRANCH}`                                                   |
+| PULL_REQUEST_BODY  | String  |  false   | Content with description for your automated pull. Default is: `"This is an automatic Pull Request to keep ${DESTINATION_BRANCH} up to date with ${SOURCE_BRANCH}! 🔄"` |
+| PULL_REQUEST_LABEL | String  |  false   | Label for your automated pull. Default is: `""`                                                                                                                        |
```

Comment:
[Nothing change visually in readme.](https://github.com/sbsrnt/sync-branch-action#inputs)
WebStorm table formatting:

![image](https://user-images.githubusercontent.com/19862545/203278529-7fa2f572-ebcf-4288-96c9-47b2b63cd211.png)